client-base/src/test/java/app/cash/backfila/embedded/JavaChangeCaseTestBackfill.java

```diff
@@ -24,9 +24,9 @@ public JavaChangeCaseTestBackfill(){
   }
 
   @NotNull @Override
-  public ValidateResult<CaseParameters> checkBackfillConfig(@NotNull BackfillConfig<CaseParameters> backfillConfig) {
-    seenParameters = backfillConfig.getParameters();
-    return super.checkBackfillConfig(backfillConfig);
+  public ValidateResult<CaseParameters> checkParameters(@NotNull CaseParameters parameters) {
```

Comment:
the only downside to this change is maybe in some cases you'd want certain parameters only in a dryrun/wetrun? probably unlikely though

Reply:
oh was this done only in this test backfill? nvm then

Reply:
yeah exactly. Initially I also sweated. Then noticed it was only for test. :)